src/amchart/CommonSerial.js

```diff
@@ -288,6 +291,7 @@
                 this._chart.valueAxes[0].logarithmic = false;
                 if (this.sortDates()) {
                     this.data(Utility.naturalSort(this.data(), "ascending", 0));
+                    this._dataUpdated--; // decrament the counter becuase its not a true data update
```

Comment:
@GordonSmith i could use your 2cents here ... im doing a data update but is there a way i can just change the underlying without calling it with this.data(x); can i use do _data? here without any consequence?